src/hooks/useConditionalNav.js

```diff
@@ -1,6 +1,7 @@
 import {useEffect} from 'react';
 import {useNavigate} from 'react-router-dom';
 
+// No references of this. Should we delete?
```

Comment:
Ref at Layout.js to navigate user right from empty page to userpage
![Screenshot_436](https://user-images.githubusercontent.com/117737662/221512186-ebfe44c8-02a0-417f-a9ec-c8fd60d2cbc4.png)

Reply:
Should we just place <Navigate to="/userpage" /> or <Redirect to="/userpage" /> into Home page instead?

Reply:
then we are not able to use nav links in the header. Everyone would redirect at userpage but i would try one more time